blockchain/api.go

```diff
@@ -88,6 +91,17 @@ type TestTokenAPI interface {
 	GetTokens(ctx context.Context, key *ecdsa.PrivateKey) (*types.Transaction, error)
 }
 
+type SimpleGatekeeperAPI interface {
+	PayIn(ctx context.Context, key *ecdsa.PrivateKey, value *big.Int) <-chan error
+	Payout(ctx context.Context, key *ecdsa.PrivateKey, to common.Address, value, txNumber *big.Int) <-chan error
+}
+
+type OracleUSDAPI interface {
```

Comment:
You define an interface for the `Oracle` entity, but specify (via naming) that this interface implementation must manipulate only the USD currency. Is there any meaningful difference in a **interface**, if we want to receive the price in EUR?